docs/sdk/app-development.md

```diff
@@ -149,20 +149,53 @@ Once your Function has been deployed, you can check its status by running:
 
         $ sls status
 
-## Local testing
+## Running Locally
 
-You can test your function locally by doing the following:
+Running functions locally is extremely simple using Google's [functions-framework](https://cloud.google.com/functions/docs/functions-framework).
 
-        $ npm install -g @google-cloud/functions-emulator
+        $ npm install @google-cloud/functions-framework
```

Comment:
The same is repeated in line 160 so I think we can remove this line.

Reply:
thanks for the catch